light_dark_mode/public/index.css

```diff
@@ -0,0 +1,145 @@
+* {
+    box-sizing: border-box;
+}
+
+h1, h2, h3, h4, h5, h6, p {
+    margin: 0;
+}
+
+body {
+    margin: 0;
+}
+
+.container {
+    display: flex;
+    flex-direction: column;
+    height: 100vh;
+}
+
+/* NAV styles */
+
+nav {
+    display: flex;
+    align-items: center;
+    background-color: #ffffff;
```

Comment:
Let's keep this order https://css-tricks.com/poll-results-how-do-you-order-your-css-properties/ of css properties. In general, the most important css properties should be on top so that it is easier to read. Let's keep this order:

1. Positioning 
position, z-index, top, right;

2. Display & Box Model
display,  all the flex properties like align-itmes or justify-content, overflow, box-sizing, width, height,  padding, border,  margin

3. Color
background, color
 
4. Text
font-family,  font-size,  line-height, text-align

5. Other
cursor

So in this nav, the order should be:

```
nav {
  z-index: 1;
  display: flex;
  height: 60px;
  padding: 30px 25px;
  align-items: center;
  background-color: #ffffff;  
  box-shadow: 0px 1px 4px rgba(0, 0, 0, 0.25);
}
```
Please apply this order to all the css classes.
 
You could save this order somewhere (i.e. to your github page) as you will need it in future when writing css code.